Gems/Terrain/Code/Source/TerrainRenderer/TerrainClipmapManager.h

```diff
@@ -123,6 +129,8 @@ namespace Terrain
         void Initialize(AZ::Data::Instance<AZ::RPI::ShaderResourceGroup>& terrainSrg);
         bool IsInitialized() const;
         void Reset();
+        //! Reset the first refresh clipmap flag so that all the clipmap image will be repainted.
+        void ResetFullRefreshClipmaps();
```

Comment:
1. Was the comment supposed to be "Reset the full refresh..." instead of "Reset the first refresh..."?
2. This function name's wording is kind of strange. Maybe something like `TriggerFullRefresh()`?
3. Does this function need to be public? it looks like it's only used privately right now.

Reply:
I'll change the name. It is public because I think there should be some API to refresh the whole map other than recreating all images.